prometheus-ksonnet/lib/alertmanager.libsonnet

```diff
@@ -17,20 +17,51 @@
     ]
   else [],
 
+  slackAlertTitle:: '{{ template "__alert_title" . }}',
+  slackAlertText:: '{{ template "__alert_text" . }}',
+
+  build_slack_receiver(name, slack_channel)::
+    {
+      name: name,
+      slack_configs: [{
+        api_url: $._config.slack_url,
+        channel: slack_channel,
+        send_resolved: true,
+        title: $.slackAlertTitle,
+        text: $.slackAlertText,
+        actions: [
+          {
+            type: 'button',
+            text: 'Runbook :green_book:',
+            url: '{{ (index .Alerts 0).Annotations.runbook_url }}',
```

Comment:
A thought: what happens when there is no runbook url on the alert?

Reply:
I have verified it with our alerts which do not have a runbook url, the button simply does not show up. I guess Slack takes care of it when URL is empty.

Reply:
Nice, thanks.